test/integration/apiserver/apply/apply_crd_test.go

```diff
@@ -129,6 +130,50 @@ spec:
 		t.Fatalf("failed to apply object with force after updating replicas: %v:\n%v", err, string(result))
 	}
 	verifyReplicas(t, result, 1)
+
+	// Try to set managed fields using a subresource and verify that it has no effect
```

Comment:
Is there a CRD test that tries to set the managed fields via the main resource? I'd like to see that that still works.

Reply:
What do you mean by "main resource"?

Reply:
Ah I see, you mean without the subresource path. 🤔 yeah, I can't see a test that explicitly tries and set the managed fields using CRDs.

Reply:
Added a new test here: https://github.com/kubernetes/kubernetes/pull/91993/commits/d9a577f73949bdd75bd8d0b2c5c1bd3fe192d1c6